test/TestVersioned.py

```diff
@@ -96,6 +96,36 @@ def test_ForMostRecent(self, requests):
         query = requests.getQs()
         self.assertEqual(query["fillerDV"],"10")
 
+    @helpers.TrackRequests
+    def test_ForMUVDateTime(self, requests):
+        url = qs.createVersioned() \
+            .forMarketData([100000001]) \
+            .inAbsoluteDateRange("2021-09-22","2021-09-23") \
+            .inTimeZone("CET") \
+            .inGranularity(Granularity.Day) \
+            .forMUV() \
+            .withFillCustomValue(10) \
+            .execute()
+
+        path = requests.getPath()
+        self.assertEqual(path[5],"M")
+        self.assertEqual(path[6],"u")
+        self.assertEqual(path[7],"v")
+
+    @helpers.TrackRequests
+    def test_ForMUVVerionLimit(self, requests):
+        url = qs.createVersioned() \
+            .forMarketData([100000001]) \
+            .inAbsoluteDateRange("2021-09-22","2021-09-23") \
+            .inTimeZone("CET") \
+            .inGranularity(Granularity.Day) \
+            .forMUV("2021-09-22T10:00") \
```

Comment:
This is misleading as the format isn't right. The API would reject this as isn't a complete datetime. is missing seconds.